libraries/AP_Notify/Display.cpp

```diff
@@ -312,46 +371,40 @@ void Display::update()
 
     timer = 0;
 
-    // check if status has changed
-    if (_flags.armed != AP_Notify::flags.armed) {
-        update_arm();
-        _flags.armed = AP_Notify::flags.armed;
-    }
-
-    if (_flags.pre_arm_check != AP_Notify::flags.pre_arm_check) {
-        update_prearm();
-        _flags.pre_arm_check = AP_Notify::flags.pre_arm_check;
-    }
-
-    if (_flags.gps_status != AP_Notify::flags.gps_status) {
-        update_gps();
-        _flags.gps_status = AP_Notify::flags.gps_status;
-    }
-
-    if (_flags.gps_num_sats != AP_Notify::flags.gps_num_sats) {
-        update_gps_sats();
-        _flags.gps_num_sats = AP_Notify::flags.gps_num_sats;
-    }
+	if (AP_Notify::flags.armed)
+	 	{
+		if (screenpage != 1)
+			{
+				clear_screen();
+				update_arm(3);
+				screenpage=1;
+				hw_update(); //update hw once , do not transmition to display in fly
```

Comment:
when plane or copter in fly onboard display do not need,  show only "ARMED" and stop font generation and bus transferring (for save CPU and BUS utilization)